src/format.ts

```diff
@@ -55,13 +58,21 @@ export function formatNumber(input: number, givenOptions: FormatOptions) {
     .trim();
 }
 
-export function formatNumberSeparatorAndDelimiter(input: number, givenOptions: FormatOptions): string {
+export function formatCurrency(input: number, givenOptions: FormatOptions = {}) {
+  return formatNumber(input, { trimTrailingZeros: false, ...givenOptions });
+}
+
+export function formatNumberSeparatorAndDelimiter(input: number, givenOptions: FormatOptions = {}): string {
   const options = mergeOptionsWithDefault(givenOptions);
-  const numbers = input.toString().split('.');
+  const numbers = input.toFixed(givenOptions.precision).toString().split('.');
 
   numbers[0] = numbers[0].replace(/\B(?=(\d{3})+(?!\d))/g, options.delimiter);
 
-  return numbers.join(options.separator);
+  const ret = numbers.join(options.separator);
+  if (givenOptions.trimTrailingZeros) {
+    return ret.replace(/0+$/g, '');
```

Comment:
I've found different problem :) Now there is trailing dot after whole numbers :-D